go/image.bzl

```diff
@@ -39,10 +39,10 @@ load(":go.bzl", "DIGESTS")
 def repositories():
     """Import the dependencies of the go_image rule.
 
-    Call the core "repositories" function to reduce boilerplate. This is
+    Call the core "go_deps" function to reduce boilerplate. This is
     idempotent if folks call it themselves.
     """
-    _repositories()
+    _go_deps()
```

Comment:
Do you need both repositories from repositories.bzl and go_repositories.bzl?

Reply:
container_repositories must be called from WORKSPACE directly (updated readme) for all image rules now